### PR TITLE
Replace "summer" with "mid-"

### LINKED ISSUE #66: Section 5.1 "summer of 2018"
In section 5.1, you write "summer of 2018".  I think this is northern-hemisphere-centric, and perhaps should list a range of months, or second-semester, or mid-2018 (to give a few examples).

### PR DESCRIPTION
**Pull request by pllim**
Fix #66 

p.s. ~~June~~ "mid" is obtained from timeline at https://github.com/astropy/astropy-APEs/blob/master/APE10.rst

p.p.s. (Angry fist at Earth's tilt)

### CODE REVIEW

File: main.tex [hunk 1]
```diff
@@ -1178,7 +1178,7 @@ \subsection{Future versions of the \astropypkg core and affiliated packages}
 will simplify the code base, and reduce the testing overhead for the package.
 \astropypkg version 3.0 is currently scheduled for January 2018.
 
-In the next major release after version 3.0, scheduled for the summer of
+In the next major release after version 3.0, scheduled for around June
```
Comment (bsipocz) [inline main.tex]: I deliberately didn't say the month to keep the paper accurate even in a case of a likely release delay.

Comment (crawfordsm) [reply main.tex]: I think 'mid-2018' would be the most appropriate

**Comment by pllim** [comment]
Thanks for the review. I believe I have addressed the comments.

**Comment by adrn** [comment]
I would just say "mid-2018" instead of "around mid-2018"

**Comment by pllim** [comment]
I thought "around" would provide even more flexibility but I guess I can remove it.

**Comment by adrn** [comment]
🃏

**Comment by adrn** [comment]
LGTM

**Pull request merged**